packages/esm-patient-registration-app/src/widgets/display-photo.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DisplayPatientPhoto from './display-photo.component';
+import { mockPatient } from '../../../../__mocks__/appointments.mock';
+
+jest.mock('../patient-registration/patient-registration.resource', () => ({
+  usePatientPhoto: jest.fn().mockReturnValue({ data: { imageSrc: 'test-image-src' } }),
```

Comment:
```suggestion
  usePatientPhoto: jest.fn(),
```
Shall we keep it as above, and mock the return value only when we are using it?

Reply:
it is getting used in both the test so we need it otherwise both the test will fail.